terraform/deployments/govuk-publishing-infrastructure/main.tf

```diff
@@ -34,3 +38,9 @@ provider "aws" {
 }
 
 provider "random" {}
+
+# used by the fastly ip ranges provider.
+# an API key is needed but 'fake' seems to work.
```

Comment:
cool 😅 (I guess there's a bit of a bug in the provider that it requires it for some calls where Fastly doesn't 🤷)